Emby.Naming/Common/NamingOptions.cs

```diff
@@ -368,6 +368,20 @@ public NamingOptions()
                     IsOptimistic = true,
                     IsNamed = true
                 },
+
+                // Series and season only expression
+                // "the show/season 1", "the show/s01"
+                new EpisodeExpression(@"(.*(\\|\/))*(?<seriesname>.+)\/[Ss](eason){0,1}[\. _\-]*(?<seasonnumber>\d+)")
```

Comment:
We should always refrain from using `\d` as it covers a whole lot of unicode digits as well.
```suggestion
                new EpisodeExpression(@"(.*(\\|\/))*(?<seriesname>.+)\/[Ss](eason){0,1}[\. _\-]*(?<seasonnumber>[0-9]+)")
```

Reply:
Fixed